pkg/pillar/rootfs/sos.sh

```diff
@@ -5,27 +5,27 @@ repeat=-1
 
 if [ $# != 0 ]; then
```

Comment:
please let's delete sos.sh instead of fixing it. (that was a previous comment of mine that appears to have gotten lost)

Reply:
I will delete.